docker_templates/alpine-linux.Dockerfile.j2

```diff
@@ -6,16 +6,13 @@ FROM {{ base_image }}
 
 RUN set -eux; \
     apk add --no-cache \
-        # bash is required for the entrypoint script
-        # see https://github.com/adoptium/containers/issues/415
-        bash \
-        # fontconfig and ttf-dejavu added to support serverside image generation by Java programs
+        # java.lang.UnsatisfiedLinkError: libfontmanager.so: libfreetype.so.6: cannot open shared object file: No such file or directory
+        # java.lang.NoClassDefFoundError: Could not initialize class sun.awt.X11FontManager
+        # https://github.com/docker-library/openjdk/pull/235#issuecomment-424466077
         fontconfig ttf-dejavu \
-        # java-cacerts added to support adding CA certificates to the Java keystore
-        java-cacerts \
-        # fixes issues with apk del apk-tools
-        # see https://github.com/adoptium/containers/issues/136
-        libretls zlib \
+        # utilities for keeping Alpine and OpenJDK CA certificates in sync
+        # https://github.com/adoptium/containers/issues/293
+        ca-certificates p11-kit-trust \
```

Comment:
Were these previously missing so the functionality didn't work, or were they pulled in implicitly by the now-removed `java-cacerts` dependency?

Reply:
Ah ok "yes" is the answer based on https://github.com/adoptium/containers/issues/458#issuecomment-1785160571